lib/dpl/provider/surge.rb

```diff
@@ -0,0 +1,33 @@
+module DPL
+  class Provider
+    class Surge < Provider
+      npm_g 'surge'
+
+      def project
+        File.expand_path("./" + (options[:project] || '') )
```

Comment:
No - if working directory changes during the build this will not work. 

I had the impression that working directory would reset to repo root when deploying. This might not be the case. 

Are you aware of how to identify repo root?